frontend/lib/analytics/google-analytics.tsx

```diff
@@ -283,14 +283,23 @@ type OutboundLinkProps = DetailedHTMLProps<
 > & {
   /** The "href" prop is required on outbound links, not optional. */
   href: string;
+  target: string;
+  rel: string;
```

Comment:
So what you're doing here is actually making `target` and `rel` _required_ props of the component, rather than optional (which is actually the default, inherited from the type a few lines above this).   Because you're making them required, [our continuous integration suite is failing](https://app.circleci.com/pipelines/github/JustFixNYC/tenants2/1775/workflows/db7ca91a-d2ae-45d8-a2ce-5d44661dd9c9/jobs/6078/steps).  You can reproduce this on your end by running `yarn typecheck`, but also, if you open any of the files with errors in VS Code, there should be big squiggly red underlines below the offending code (let me know if there aren't).